settings.gradle.kts

```diff
@@ -3,13 +3,13 @@ pluginManagement {
     mavenLocal()
     gradlePluginPortal()
     maven {
-      url = uri("https://dl.bintray.com/hypertrace/maven")
+      url = uri("https://hypertrace.jfrog.io/artifactory/gradle")
```

Comment:
Is there anything specific to `gradle`? Could have been `https://hypertrace.jfrog.io/artifactory/maven` as we have some maven-related metadata xml/pom files under the project.

Reply:
updated artifactory url.

Reply:
yes there are both specific gradle and maven artifacts stored here, since the repo format can be used for either - I intentionally named it gradle to avoid confusion since that's the tool we're using, but it's not worth the effort to change back since this already went in.